module/product/src/Resources/translations/attribute.en.yaml

```diff
@@ -0,0 +1 @@
+PRODUCT-RELATION: Relation
```

Comment:
Product relation? Do not know whether it's worth being that exact

Reply:
IMHO no.